_ml-commons-plugin/agents-tools/tools/connector-tool.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: default
+title: Connector tool
+has_children: false
+has_toc: false
+nav_order: 20
+parent: Tools
+grand_parent: Agents and tools
+---
+
+<!-- vale off -->
+# Connector tool
+**Introduced 2.15**
+{: .label .label-purple }
+<!-- vale on -->
+
+The `ConnectorTool` runs `execute` action in a connector.
+
+## Step 1: Register a connector with execute action
+
+`ConnectorTool` can only run `execute` action in a connector. So you need to create a connector with `execute` action. It's very similar to the `predict` action in connector. For example, create a connector to execute an AWS Lambda function with [Function URL](https://docs.aws.amazon.com/lambda/latest/dg/lambda-urls.html).
```

Comment:
it will be helpful to link the page to explain connector. https://opensearch.org/docs/latest/ml-commons-plugin/remote-models/connectors/

Reply:
sure